packages/composables/src/useCart.ts

```diff
@@ -189,9 +189,7 @@ export function useCartFunction(): UseCartReturn {
   const count = computed(() => {
     return cartItems.value.reduce(
       (accumulator: number, lineItem: LineItem) =>
-        lineItem.type === "product"
-          ? lineItem.quantity + accumulator
-          : accumulator,
+        lineItem.good === true ? lineItem.quantity + accumulator : accumulator,
```

Comment:
type - Type refers to the entity type of an item whether it is product or promotion for instance.

good - When set to true, it indicates the line item is physical else it is virtual.


Are you sure about this change? Looks like the virtual product will be omitted

Reply:
thanks for hinting that! I've created an issue regarding it IMO in a complex way: https://github.com/shopware/frontends/issues/456

regarding the `good` thing ;) do we have such thing like virtual product? I know the digital one, and the promotion which is now displayed when there is no product in the cart. do we maybe want to show any promotion anyway?